app/services/ingest_iati_activities.rb

```diff
@@ -154,14 +158,14 @@ def call
       providing_organisation_reference = transaction_element.children.detect { |child| child.name.eql?("provider-org") }.attributes["ref"].value
       receiving_organisation_name = transaction_element.children.detect { |child| child.name.eql?("receiver-org") }.children.detect { |child| child.name.eql?("narrative") }.text
       receiving_organisation_reference = transaction_element.children.detect { |child| child.name.eql?("receiver-org") }.attributes["ref"].try(value, nil)
-
+# binding.pry
```

Comment:
😱

Reply:
🤭

Reply:
@lozette I've removed the pry, thanks for spotting.